lib/FilesHooks.php

```diff
@@ -793,17 +793,28 @@ protected function unshareFromUser(IShare $share) {
 			return;
 		}
 
+		if ($share->isExpired()) {
+			// Link expired
+			$actionSharer = 'expired_user_by';
```

Comment:
actually I think the `_by` is confusing and should be removed. It's not expired by the user, but by time...

Reply:
This is true for all expire messages

Reply:
Good to me!